hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -498,9 +498,14 @@
   public static final int
       DFS_NAMENODE_SNAPSHOT_DIFF_LISTING_LIMIT_DEFAULT = 1000;
 
-  public static final String DFS_NAMENODE_SNAPSHOT_MAX_LIMIT =
-      "dfs.namenode.snapshot.max.limit";
+  public static final String
+      DFS_NAMENODE_SNAPSHOT_MAX_LIMIT = "dfs.namenode.snapshot.max.limit";
```

Comment:
Lets change DFS_NAMENODE_SNAPSHOT_MAX_LIMIT to DFS_NAMENODE_SNAPSHOT_MAX_DIRECTORY_LIMIT. and lets use the same reference of the variable internally;

Reply:
It was changed back to the original name as per Nicholas suggestion.

Reply:
sure